lib/phlex/parent_node.rb

```diff
@@ -5,12 +5,11 @@ module ParentNode
     include Callable
 
     def children
-      @_children ||= []
+      @_children ||= +""
```

Comment:
I wonder if we should rename this `@_content` instead. 🤔

Reply:
Yes, I think it would make more sense to rename this now :)

Reply:
Can’t rename to "content" because it's taken by the `content(&)` method. I’m going to merge as is for now.